action.yml

```diff
@@ -0,0 +1,202 @@
+---
+name: Steampipe Check
+author: Turbot
+description: Run Steampipe mod controls and create annotations for Infrastructure as Code controls.
+branding:
+  color: red
+  icon: shield
+inputs:
+  mod-url:
+    description: "Mod URL to clone. Examples: https://github.com/turbot/steampipe-mod-aws-compliance"
+    required: true
+  mod-branch:
+    description: "Mod branch or tag to clone. Defaults to 'main'. Examples: main, release/v0.7, 0.5, 1.0."
+    required: false
+    default: main
+  checks:
+    description: Space separated list of benchmarks and controls to run. Can also pass in as a multi-line string. Defaults to 'all'.
+    required: false
+    default: all
+  artifact-exports:
+    description: "Comma separated export types to save as artifacts. Example: 'json,csv'. Defaults to 'csv'. Valid values: 'asff,csv,html,json,md,nunit13,sps'."
+    required: false
+    default: csv
```

Comment:
I can't recall, did we agree to keep `csv` as the default since it was a low file size?

Also, is there a way for a user to not create any artifacts?

Reply:
Yes, I believe you were interested in keeping CSV as a minimum artifact.

Yes, they can override and pass nothing: [example](https://github.com/SektorNine/testasaurus/actions/runs/6309645803) output.

Reply:
Make fully opt-in, remove the default.